### ISSUE TITLE
Unable to install via pip/mamba/conda on Windows 10

### ISSUE THREAD

**Issue by Karlheinzniebuhr** [DLR-RM/stable-baselines3#1304]
### 🐛 Bug

I'm trying to install stable-baselines3 via conda but it fails as it can't resolve the dependencies.

### To Reproduce

```python
mamba install -c conda-forge stable-baselines3
```

```
Looking for: ['stable-baselines3']

conda-forge/win-64                                          Using cache
conda-forge/noarch                                          Using cache

Pinned packages:
  - python 3.11.*


Could not solve for environment specs
Encountered problems while solving:
  - nothing provides pytorch >=1.4.0 needed by stable-baselines3-1.0-pyhd8ed1ab_0

The environment can't be solved, aborting the operation
```

`conda install -c conda-forge stable-baselines3`

```
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: /
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed

UnsatisfiableError: The following specifications were found to be incompatible with each other:

Output in format: Requested package -> Available versions

PS C:\dev\Python\Mastermind\mastermind> python -c 'import stable_baselines3 as sb3; sb3.get_system_info()'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'stable_baselines3'
```

### Checklist

- [X] I have checked that there is no similar [issue](https://github.com/DLR-RM/stable-baselines3/issues) in the repo
- [X] I have read the [documentation](https://stable-baselines3.readthedocs.io/en/master/)
- [X] I have provided a minimal working example to reproduce the bug
- [X] I've used the [markdown code blocks](https://help.github.com/en/articles/creating-and-highlighting-code-blocks) for both code and stack traces.

**Comment by Karlheinzniebuhr** [comment]
pip also fails as follows.

`pip install stable-baselines3`

```
Collecting stable-baselines3
  Using cached stable_baselines3-1.7.0-py3-none-any.whl (171 kB)
Collecting gym==0.21
  Downloading gym-0.21.0.tar.gz (1.5 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.5/1.5 MB 2.4 MB/s eta 0:00:00
  Preparing metadata (setup.py) ... done
Collecting numpy
  Downloading numpy-1.24.1-cp311-cp311-win_amd64.whl (14.8 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 14.8/14.8 MB 2.4 MB/s eta 0:00:00
Collecting stable-baselines3
  Downloading stable_baselines3-1.6.2-py3-none-any.whl (170 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 170.0/170.0 kB 2.0 MB/s eta 0:00:00
  Downloading stable_baselines3-1.6.1-py3-none-any.whl (180 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 180.0/180.0 kB 1.8 MB/s eta 0:00:00
  Downloading stable_baselines3-1.6.0-py3-none-any.whl (177 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 177.2/177.2 kB 1.8 MB/s eta 0:00:00
  Downloading stable_baselines3-1.5.0-py3-none-any.whl (177 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 177.7/177.7 kB 1.8 MB/s eta 0:00:00
  Downloading stable_baselines3-1.4.0-py3-none-any.whl (176 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 176.9/176.9 kB 1.8 MB/s eta 0:00:00
Collecting gym<0.20,>=0.17
  Downloading gym-0.19.0.tar.gz (1.6 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.6/1.6 MB 1.9 MB/s eta 0:00:00
  Preparing metadata (setup.py) ... done
Collecting stable-baselines3
  Downloading stable_baselines3-1.3.0-py3-none-any.whl (174 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 174.7/174.7 kB 2.1 MB/s eta 0:00:00
  Downloading stable_baselines3-1.2.0-py3-none-any.whl (161 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 162.0/162.0 kB 1.9 MB/s eta 0:00:00
Collecting gym>=0.17
  Using cached gym-0.26.2.tar.gz (721 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Collecting stable-baselines3
  Downloading stable_baselines3-1.1.0-py3-none-any.whl (172 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 172.6/172.6 kB 2.1 MB/s eta 0:00:00
  Downloading stable_baselines3-1.0-py3-none-any.whl (152 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 152.7/152.7 kB 2.3 MB/s eta 0:00:00
  Downloading stable_baselines3-0.11.1-py3-none-any.whl (152 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 152.2/152.2 kB 1.8 MB/s eta 0:00:00
  Downloading stable_baselines3-0.11.0-py3-none-any.whl (152 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 152.1/152.1 kB 1.8 MB/s eta 0:00:00
  Downloading stable_baselines3-0.10.0-py3-none-any.whl (145 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 146.0/146.0 kB 1.7 MB/s eta 0:00:00
  Downloading stable_baselines3-0.9.0-py3-none-any.whl (128 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 128.8/128.8 kB 2.5 MB/s eta 0:00:00
  Downloading stable_baselines3-0.8.0-py3-none-any.whl (128 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 128.6/128.6 kB 2.5 MB/s eta 0:00:00
  Downloading stable_baselines3-0.7.0-py3-none-any.whl (110 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 110.8/110.8 kB 2.1 MB/s eta 0:00:00
  Downloading stable_baselines3-0.6.0-py3-none-any.whl (105 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 105.6/105.6 kB 2.0 MB/s eta 0:00:00
ERROR: Cannot install stable-baselines3==0.10.0, stable-baselines3==0.11.0, stable-baselines3==0.11.1, stable-baselines3==0.6.0, stable-baselines3==0.7.0, stable-baselines3==0.8.0, stable-baselines3==0.9.0, stable-baselines3==1.0, stable-baselines3==1.1.0, stable-baselines3==1.2.0, stable-baselines3==1.3.0, stable-baselines3==1.4.0, stable-baselines3==1.5.0, stable-baselines3==1.6.0, stable-baselines3==1.6.1, stable-baselines3==1.6.2 and stable-baselines3==1.7.0 because these package versions have conflicting dependencies.

The conflict is caused by:
    stable-baselines3 1.7.0 depends on torch>=1.11
    stable-baselines3 1.6.2 depends on torch>=1.11
    stable-baselines3 1.6.1 depends on torch>=1.11
    stable-baselines3 1.6.0 depends on torch>=1.11
    stable-baselines3 1.5.0 depends on torch>=1.8.1
    stable-baselines3 1.4.0 depends on torch>=1.8.1
    stable-baselines3 1.3.0 depends on torch>=1.8.1
    stable-baselines3 1.2.0 depends on torch>=1.8.1
    stable-baselines3 1.1.0 depends on torch>=1.4.0
    stable-baselines3 1.0 depends on torch>=1.4.0
    stable-baselines3 0.11.1 depends on torch>=1.4.0
    stable-baselines3 0.11.0 depends on torch>=1.4.0
    stable-baselines3 0.10.0 depends on torch>=1.4.0
    stable-baselines3 0.9.0 depends on torch>=1.4.0
    stable-baselines3 0.8.0 depends on torch>=1.4.0
    stable-baselines3 0.7.0 depends on torch>=1.4.0
    stable-baselines3 0.6.0 depends on torch>=1.4.0

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts
```

**Comment by araffin** [comment]
Hello,
please install PyTorch first in your case.

please also fill the issue template completely (system info), looking at the backtracking, i'm suspecting that you are using outdated pip version too.

**Comment by araffin** [comment]
oh, i see, you are using python 3.11, you must use the PyTorch nightly version.

**Comment by Karlheinzniebuhr** [comment]
You are right it was a pytorch problem, solved by downgrading to python 3.9
